### PR TITLE
412025 - Add prominent links to the #ux-aspects channel

### PR DESCRIPTION
**Pull request by JonathanToddMF**
#### Checklist

* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licensed under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue

https://internal.almoctane.com/ui/entity-navigation?p=131002/7002&entityType=work_item&id=412025

#### Description of Proposed Changes

- Removed blog and blog assets
- Added 1 column option to feature list
- Added link in navbar
- Removed links to blog and twitter

#### Documentation CI URL

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_412025-prominent-link-to-teams-channel

#### Downstream Build(s)

* [caf/ux-aspects-micro-focus](https://jenkins.swinfra.net/job/SEPG/job/caf/job/caf~ux-aspects-micro-focus~412025-prominent-link-to-teams-channel~CI/)

### CODE REVIEW

**Comment by JonathanToddMF** [comment]
Hi @Roland-Penny as discussed here are the 3 examples I came up with for the new Get Involved section

Replaced with new colourful icons
screencapture-localhost-1

Replaced colourful icons smaller
screencapture-localhost-2

No icons
screencapture-localhost-3

Personally I think the first one is fine with the icons being the same size as the others. 

@donna-marijne do you have any opinion on this? Roland and I spoke about replacing the twitter icon with a similar Teams icon but unfortunately their icon uses multiple colours and isn't one solid colour. So we discussed replacing these icons with the colourful variants.

**Review by donna-marijne — CHANGES_REQUESTED** [review]

File: docs/app/components/navigation-bar/navigation-bar.component.html [hunk 1]
```diff
@@ -23,6 +23,15 @@
...
         </a>
 
+        <ng-container *ngIf="themes">
+            <a class="navigation-bar-item"
+               *ngFor="let link of help"
+               [attr.href]="link.link"
+               routerLinkActive="active">
+               {{ link.title }}
+            </a>
+        </ng-container>
```
[the remark this answers is not in the document]
Comment (JonathanToddMF) [reply docs/app/components/navigation-bar/navigation-bar.component.html:33]: Makes sense, is there a way to have an external url with the routerLink? I was having some trouble getting it to work.

[the remark this answers is not in the document]
Comment (donna-marijne) [reply docs/app/components/navigation-bar/navigation-bar.component.html:33]: I would suggest extending the JSON schema with an optional `href` property which can be used for external links.

[the remark this answers is not in the document]
Comment (JonathanToddMF) [reply docs/app/components/navigation-bar/navigation-bar.component.html:33]: `<a class="navigation-bar-item"
           *ngFor="let link of social"
           [routerLink]="link.link"
           [attr.href]="link.href"
           routerLinkActive="active">
           {{ link.title }}
</a>`

Was kind of hoping the following would work but there doesn't seem to be a way to conditionally turn off the routerLink. Unless you have any ideas? Maybe a custom directive but unsure if that is worth it for this implementation?

[the remark this answers is not in the document]
Comment (donna-marijne) [reply docs/app/components/navigation-bar/navigation-bar.component.html:33]:
```html
<a *ngIf="link.link" [routerLink]="link.link" ...>{{ link.title }}</a>
<a *ngIf="!link.link" [href]="link.href" ...>{{ link.title }}</a>
```

Unfortunately the Angular directives are badly designed in this regard, and force you to duplicate code.

File: docs/app/components/navigation-bar/navigation-bar.component.ts [hunk 2]
```diff
@@ -13,9 +13,10 @@ export class NavigationBarComponent {
     navigation = this._appConfig.getConfigurationData('top-navigation');
     brand: ILink = this.navigation.brand;
     links: ILink[] = this.navigation.links;
+    help: ILink[] = this.navigation.help;
```
Comment (donna-marijne) [inline docs/app/components/navigation-bar/navigation-bar.component.ts:16]: Not seeing where this is used.

**Review by donna-marijne — CHANGES_REQUESTED** [review]
Thanks, it looks good, code needs a bit of a cleanup.

**Review by donna-marijne — APPROVED** [review]
Verification:

Blog and Twitter link removed:
https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_412025-prominent-link-to-teams-channel/#/landing - :heavy_check_mark:

Blog and Twitter link removed; MS Teams links added:
https://pages.github.houston.softwaregrp.net/sepg-docs-qa/caf_CI_ux-aspects-micro-focus_412025-prominent-link-to-teams-channel/#/landing - :heavy_check_mark:

**Review by Roland-Penny — APPROVED** [review]
Looks good

**Pull request merged**